infra/periodic_killer.tf

```diff
@@ -70,7 +70,7 @@ CRON
 chmod +x /root/periodic-kill.sh
 
 cat <<CRONTAB >> /etc/crontab
-*/30 * * * * root /root/periodic-kill.sh
+0 4 * * * root /root/periodic-kill.sh
```

Comment:
Does that mean we're running the job 4 am UTC or local time?

Reply:
UTC.